tests/functional/centos/7/cluster/vagrant_variables.yml

```diff
@@ -10,7 +10,7 @@ mds_vms: 1
 rgw_vms: 1
 nfs_vms: 0
 rbd_mirror_vms: 0
-client_vms: 0
+client_vms: 1
```

Comment:
Alright, then that I did :). I'll do what you suggest for `xenial_cluster`. Thanks.